### PR TITLE
fix ohlc

### PR DESCRIPTION
**Pull request by 4p00rv**
Fix for ohlc bar not getting updated properly.

### CODE REVIEW

File: src/websockets/stream_handler.js [hunk 1]
```diff
@@ -136,31 +128,28 @@ define(["websockets/binary_websockets", "charts/chartingRequestMap", "common/uti
...
                     if (isNew) {
                         series.addPoint([time, close], true, true, false);
                     } else {
-                        last.update({
-                            y : close
-                        });
+                        last[1] = close;
+                        chart.series[0].options.data[len] = last;
+                        // Redrawing the chart to update new point.
+                        chart.series[0].setData(chart.series[0].options.data, true);
```
Comment (arnabk) [inline src/websockets/stream_handler.js]: @4p00rv Is this going to be called for every update?

Comment (4p00rv) [inline src/websockets/stream_handler.js]: @arnabk Yes, that's what the developer suggested as a fix.

Comment (arnabk) [inline src/websockets/stream_handler.js]: can you send me the link. This seems to be very bad suggestion. For every update, it is going to reset so many things and refresh all data.

Comment (4p00rv) [inline src/websockets/stream_handler.js]: Here's the link: https://github.com/highcharts/highcharts/issues/5385
Its the second last comment in which the developer suggests to use `series.setdata`, although he seems to be concerned about performance too.

**Comment by arnabk** [comment]
@4p00rv I looked at it. Its a bit risky change. We have bunch of indicators coded and they will not update if we change values like this. Highchart data update needs to trigger when we update data point in order to trigger calculation in indicators

**Pull request closed without merging**